test/jdk/jdk/incubator/vector/templates/Perf-header.template

```diff
@@ -77,6 +77,8 @@ public class $vectorbenchtype$ extends AbstractVectorBenchmark {
         return $Boxtype$.compare(a, ($type$) 0) != 0 ? a : b;
     }
 
+    private static final $type$ CONST_SHIFT = $Boxtype$.SIZE / 2;
+
```

Comment:
This is only needed for integral types and not for float/double.

Reply:
Done